astropy/io/ascii/tests/test_c_reader.py

```diff
@@ -888,3 +888,101 @@ def test_fast_tab_with_names(parallel, read_tab):
     head = ['A{0}'.format(i) for i in range(28)]
     table = read_tab(content, data_start=1,
                      parallel=parallel, names=head)
+
+
+# catch Windows environment since we cannot use _read() with custom fast_reader
+@pytest.mark.parametrize("parallel", [
+    pytest.mark.xfail(os.name == 'nt', reason=
+                      "Multiprocessing is currently unsupported on Windows")(True),
+    False])
+def test_data_out_of_range(parallel, read_basic):
+    """
+    Exponents beyond the float64 range (+-308) currently raise an error in
+    the C parser; such numbers are returned as strings instead, not so
+    with the Python parser!
+    """
+    if TRAVIS:
```

Comment:
I had only just changed it from `parallel and TRAVIS` in 4a44b6f2e752, pretty sure it did crash Travis in serial as well, but I could of course give it another try.